packages/flutter_web_plugins/lib/src/plugin_event_channel.dart

```diff
@@ -105,16 +102,16 @@ class PluginEventChannel<T> {
 }
 
 class _EventChannelHandler<T> {
-  _EventChannelHandler(this.name, this.codec, this.controller, this.messenger) : assert(messenger != null);
```

Comment:
I think we've been leaving the asserts in with an ignore, because users that are not opted into null safety can still pass null in

Reply:
Ah, very good point, I'll get those back in!